tests/common/pruning/tensor.py

```diff
@@ -22,28 +22,42 @@
 class NPNNCFTensorProcessor(NNCFBaseTensorProcessor):
```

Comment:
what does `NP` stand for? some description would be helpful

Reply:
It means that tensor uses numpy realization of a tensor, like PTNNCFTensor (Pytorch implementation) and TFNNCFTensor (TF implementation). I'll add description